pkgs/tools/backup/tsm-client/default.nix

```
@@ -126,14 +161,22 @@ let
     '';
   };
 
+  binPath = lib.strings.makeBinPath (
+    [ acl gnugrep procps ]
+    ++
+    lib.lists.optional enableGui jdk8
+  );
```

```suggestion
  binPath = lib.makeBinPath ([ acl gnugrep procps ]
    ++ lib.optional enableGui jdk8);
```
Please use lib.makeBinPath instead of lib.strings.makeBinPath.

---

This is now shortened as suggested.  However, there are many more `lib.${NAMESPACE}.${FUNCTION}` constructs around.  Do you suggest to shorten all of them?  I'm a bit reluctant to doing so as I see these namespaces to increase readability (at least for me :) .

---

They are not usually used and so far I don't know any clashing names. Doesn't matter tbh.